sources/packages/api/src-sql/Types/Create-designation-agreement-status.sql

```diff
@@ -0,0 +1,2 @@
+CREATE TYPE sims.designation_agreement_status
+  AS ENUM ('Pending', 'Approved', 'Declined');
```

Comment:
What does pending status represent? does it mean the designation requested? or not even requested?

Reply:
It means that it was requested and it is pending from Ministry review.
In the enum there are further explanation.
![image](https://user-images.githubusercontent.com/61259237/152598364-98827d65-f0a4-4b6d-a0d1-f55dee4e4d27.png)